### PR TITLE
[idea] tabbed UI changes has been done

### LINKED ISSUE #150: Tabbed UI alterations.
Move "Main Pane" to right of Code Pane and rename to "All".
Rename "Code Pane" to "Featured".

### PR DESCRIPTION
**Pull request by amit-shar**
fixes #150

### CODE REVIEW

**Comment by amit-shar** [comment]
@ScrapCodes Please review.

**Comment by ScrapCodes** [comment]
Can you put screenshots ?

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/MainWindow.java [hunk 1]
```diff
@@ -47,8 +47,8 @@
     private static final String PROJECTS = "Projects";
     protected static final String JAVA = "java";
     private static final double DIVIDER_LOCATION = 0.2;
-    private static final String MAIN_PANE = "Main Pane";
-    private static final String CODE_PANE = "Code Pane";
+    private static final String MAIN_PANE = "All";
```
Comment (ScrapCodes) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/MainWindow.java]: Rename: MAIN_PANE -> ALL_TAB

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/MainWindow.java [hunk 2]
```diff
@@ -47,8 +47,8 @@
...
+    private static final String CODE_PANE = "Featured";
```
Comment (ScrapCodes) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/MainWindow.java]: similarly FEATURED_TAB

**Pull request closed without merging**